cassiopeia/solver/MaxCutSolver.py

```diff
@@ -0,0 +1,96 @@
+"""
+"""
+import itertools
+import numpy as np
+import pandas as pd
+import networkx as nx
+from typing import Callable, Dict, List, Optional, Tuple
+
+from cassiopeia.solver import GreedySolver
+from cassiopeia.solver import graph_utils
+
+
+class MaxCutSolver(GreedySolver.GreedySolver):
+    def __init__(
+        self,
+        character_matrix: pd.DataFrame,
+        missing_char: str,
+        meta_data: Optional[pd.DataFrame] = None,
+        priors: Optional[Dict] = None,
+        sdimension: Optional[int] = 3,
+        iterations: Optional[int] = 50,
+        weights: Optional[Dict] = None
```

Comment:
Add documentation here. What is `sdimension`? What does `iterations` refer to (i.e., for what procedure)? What are the `weights`? 

Also, for `weights`, you need to specify the types that go into the dictionary.